Search placement card

georgiashay opened the pull request:
Fixes #216 

Got rid of v-menu and used v-card instead. 

Replaced updating position on size with @media.  It seems like there are only 2 sizes it is possible to get to, but maybe we should test this on a bigger screen to make sure?  If we think this is too risky we can add the repositioning back in, but I am hesitant to keep that lag.

npfoss commented on src/pages/MainPage.vue:
```diff
@@ -420,18 +411,6 @@ export default {
-
-    document.body.addEventListener('click', function (e) {
-      this.showSearch = false;
-    }.bind(this));
```
I think we want this -- closes search when click away

georgiashay commented:
Fixed problems, also added close on esc (although it only works if the input is focused, so won't work if you have dragged something from the class search and your focus isn't the search anymore).
